background.js

```diff
@@ -43,6 +43,15 @@ async function waitFor(f) {
     return f();
 };
 
+/*
+ * Check if all conditions for SSO are met
+ */
+function is_operational() {
+    if (!state_active || !accounts.active || !broker_online)
+        return false;
+    return true;
```

Comment:
Why not `return state_active && accounts.active && broker_online`?

Reply:
Well... Yes, fixed ;)